src/modules/pos/validator/components/ChangeCommission/Form/ChangeCommissionForm.js

```diff
@@ -33,16 +48,45 @@ export const ChangeCommissionForm = ({ nextStep }) => {
     });
   };
 
-  const newCommissionParam = convertCommissionToNumber(newCommission);
-  const isFormValid = newCommissionParam && newCommissionParam >= 0 && newCommissionParam <= 10000;
   const formProps = {
     moduleCommand: MODULE_COMMANDS_NAME_MAP.changeCommission,
-    params: { newCommission: newCommissionParam },
-    fields: { newCommission },
-    isFormValid,
+    params: { newCommission: newCommission.numericValue },
+    fields: { newCommission: newCommission.value, token },
+    isFormValid: newCommission.value !== currentCommission && !newCommission.feedback,
   };
   const commandParams = {
-    newCommission: newCommissionParam,
+    newCommission: newCommission.numericValue,
+  };
+
+  const checkCommissionFeedback = (value) => {
+    let inputFeedback;
+    const newCommissionParam = convertCommissionToNumber(value);
+    const newCommissionValid = checkCommissionValidity(value, currentCommission);
+    const isFormValid =
+      value !== currentCommission &&
+      newCommissionParam &&
+      newCommissionParam >= 0 &&
+      newCommissionParam <= 10000 &&
+      newCommissionValid;
+    if (checkCommissionValidity(value, currentCommission)) {
+      if (isFormValid) {
+        inputFeedback = undefined;
+      }
+    } else if (newCommissionParam >= 0 && newCommissionParam <= 10000) {
+      inputFeedback = t('You cannot increase commission more than 5%');
+    } else {
+      inputFeedback = t('Commission range is invalid');
+    }
+    return { feedback: inputFeedback, numericValue: newCommissionParam };
```

Comment:
is see duplication in the conditionals. I guess it can be refined to prevent this